templates/search.js.erb

```diff
@@ -1,121 +1,127 @@
-(function () {
-  var ARTICLES = <%= articles %>;
-  var DICTIONARY = <%= dictionary %>;
-  var MAX_RESULTS = 30;
-  var MIN_SCORE = 15;
-  var PUNCTUATION = /['";:.()?-]/g;
-  var WHITESPACE = /\s+/;
-  var rootURL = 'https://support.dnsimple.com';
+var ARTICLES = <%= articles %>;
+var DICTIONARY = <%= dictionary %>;
+var MAX_RESULTS = 30;
+var MIN_SCORE = 15;
+var PUNCTUATION = /['";.()?-]/g;
```

Comment:
I removed `:` to support our little-know "cat:dns" syntax (the original failing test).